BlockSettleUILib/ChatUI/OTCShieldWidgets/OTCNegotiationRequestWidget.cpp

```diff
@@ -283,7 +283,7 @@ void OTCNegotiationRequestWidget::onParentAboutToHide()
 void OTCNegotiationRequestWidget::onCurrentWalletChanged()
 {
    auto recvHdWallet = getCurrentHDWallet();
-   if (recvHdWallet->isHardwareWallet()) {
+   if (!recvHdWallet->canMixLeafs()) {
```

Comment:
Multiple for "leaf" is "leaves", not "leafs".

Reply:
Done, thanks